README.md

```diff
@@ -0,0 +1,169 @@
+# Lecture 05 otus Introduction with GCP (via KodeKloud Playground)
```

Comment:
This README file in root is duplicated `DocForMe/Otus05/README.md`

Duplication is bad because you need to support two copies of your README so:
1. Add only link in root README.md to `DocForMe/Otus05/README.md`. As an example check this https://github.com/loktionovam/loktionovam_platform
2. Remove all image files from your root to documentation directory
3. Also there is a convention in folder naming so I advice you to rename your `DocForMe` to `doc`